client/core/core.go

```diff
@@ -4039,16 +4039,18 @@ func (c *Core) feeSuggestion(dc *dexConnection, assetID uint32) (feeSuggestion u
 	return dc.fetchFeeRate(assetID)
 }
 
-// Withdraw initiates a withdraw from an exchange wallet. The client password
-// must be provided as an additional verification.
-func (c *Core) Withdraw(pw []byte, assetID uint32, value uint64, address string) (asset.Coin, error) {
```

Comment:
Does this mean `core.Withdraw` should send (subtract `false`) for eth to maintain existing behavior or error?

Reply:
Oh right.  The answer would be, just allow it to error.  However, I think I'm changing my mind on this since Withdraw is currently unsatisfiable for ETH.  Legacy consumers that were using Withdraw for UTXO assets can just switch to Send.

Nevermind then.